cws2/models/phono_system.py

```diff
@@ -0,0 +1,90 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.urls import reverse
```

Comment:
`reverse` is never used in this file so the line can be deleted - hints like this come up in the linter (you can run it with `make lint`)

Reply:
Yea. I saw this failed due to a linter. Thanks for setting this up, I'm a fan of linting.